src/content/manager/asset.manager.ts

```diff
@@ -55,9 +55,7 @@ export class AssetManager extends BaseManager {
     }
 
     public getBody(): any {
-        return {
-            body: JSON.stringify(this.toNodeTransport()),
-        };
+        return this.toNodeTransport();
```

Comment:
should this still use JSON.stringify?

Reply:
Axios handles json transformations automatically ([features](https://github.com/axios/axios#features)).